Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-proc_macros = { package = "grouped_ordering_proc_macros", path = "proc_macros", version = "0.1.0" }
+grouped_ordering_proc_macros = { path = "proc_macros", version = "0.1.0" }
```

Comment:
Looking at the tree-sitter-grep fully-populated collection macros as a reference, it seemed like given that this (`grouped_ordering!()`) macro is intended to be used "externally" I was going to run into trouble if there were two different names to refer to "this proc-macro crate" (`proc_macros` vs `grouped_ordering_proc_macros`)

So my understanding is this makes there no longer be a local "alias" for `grouped_ordering_proc_macros`